test/functional/tests/fault_injection/test_fault_injection_raid_as_cache.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from storage_devices.raid import Raid, RaidConfiguration, MetadataVariant, Level
+from test_tools import fs_utils
+from test_tools.disk_utils import Filesystem
+from test_utils.size import Size, Unit
+from tests.lazy_writes.recovery.recovery_tests_methods import compare_files
+
+mount_point = "/mnt/test"
+test_file_path = f"{mount_point}/test_file"
+test_file_tmp_path = "/tmp/filetmp"
+test_file_size = Size(500, Unit.KiloByte)
+files_number = 100
+
+
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.require_disk("core", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("cache1", DiskTypeSet([DiskType.sata, DiskType.hdd]))
+@pytest.mark.require_disk("cache2", DiskTypeSet([DiskType.sata, DiskType.hdd]))
+def test_fault_injection_raid_as_cache(cache_mode):
```

Comment:
I think this test should not be placed in fault injection group. Please move it to the volumes test group and rename both file and test removing "fault_injection" part